web-console/knikubevirt/expose-secret/expose-secret.md

```diff
@@ -0,0 +1,48 @@
+#Expose Secret, Config Maps and Service Accounts as Disks
+
+Kubernetes recently introduced three new volume types (Secrets, Config Maps and Service Accounts). These volumes can be used as additional VM disks, and allow better integration of Virtual Machines with Kubernetes resources.
+We will try and define where is the best place for the user to handle these disks.
```

Comment:
I think we can drop this line, at this point we've settled where the design will go.